core/src/main/java/org/apache/rocketmq/schema/registry/core/api/v1/SchemaController.java

```diff
@@ -60,13 +63,15 @@ public SchemaController(
         final RequestProcessor requestProcessor,
         final SchemaService<SchemaDto> schemaService
     ) {
+        this.cluster = QualifiedName.DEFAULT_CLUSTER;
+        this.tenant = QualifiedName.DEFAULT_TENANT;
         this.requestProcessor = requestProcessor;
         this.schemaService = schemaService;
     }
 
     @RequestMapping(
         method = RequestMethod.POST,
-        path = "/cluster/{cluster-name}/subject/{subject-name}/schema/{schema-name}",
```

Comment:
Cluster/Tenant can be default, but i think there still needs interface to expose to users. Otherwise we can only add clusters and tenants through configuration and restart.

Just like the bottom two interfaces both need exist:
/cluster/{cluster-name}/subject/{subject-name}/schema/{schema-name}
/subject/{subject-name}/schema/{schema-name}